tests/php/ORM/DataObjectSchemaGenerationTest.php

```diff
@@ -620,9 +620,9 @@ public static function provideSortFieldBecomesIndexes(): array
             'a few different field types in sort' => [
                 'defaultSort' => [
                     '"Sort"' => 'DESC',
+                    'SomeDBHtmlVarchar' => 'ASC',
                     '"SomeDBText"' => 'DESC',
                     '"SomeDBHtmlText"' => 'ASC',
-                    'SomeDBHtmlVarchar' => 'ASC',
```

Comment:
Swapping the order here so we can see that we still have multiple columns in the composite index up until the first TEXT column